dev-python/cadquery/cadquery-2.2_rc20220506.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
Can we use the new PEP517 mode here?
```
DISTUTILS_USE_PEP517=setuptools
```
(And maybe in cadquery-pywrap to)

Reply:
Tried it with localrepo and it seems to work fine. Added to the PR.